lms/static/scripts/frontend_apps/components/BookPicker.js

```diff
@@ -60,24 +59,26 @@ export default function BookPicker({ onCancel, onSelectBook }) {
   );
 
   useEffect(() => {
-    if (step === 'select-book' && !bookList) {
-      vsService.fetchBooks().then(setBookList).catch(setError);
-    } else if (step === 'select-chapter' && !chapterList) {
+    if (step === 'select-chapter' && !chapterList) {
       const currentBook = /** @type {Book} */ (book);
       vsService
         .fetchChapters(currentBook.id)
         .then(setChapterList)
         .catch(setError);
     }
-  }, [book, bookList, step, chapterList, vsService]);
+  }, [book, step, chapterList, vsService]);
 
   const canSubmit =
     (step === 'select-book' && book) || (step === 'select-chapter' && chapter);
 
   return (
     <Modal
       onCancel={onCancel}
-      title={step === 'select-book' ? 'Select book' : 'Select chapter'}
+      title={
+        step === 'select-book'
+          ? 'Paste URL to VitalSource book'
```

Comment:
We should be consistent about using the term "link" vs URL to refer to a VitalSource book link. I prefer to use "link" as that seems more friendly to me.